cognee/tests/unit/processing/chunks/chunk_by_paragraph_test.py

```diff
@@ -0,0 +1,69 @@
+from cognee.tasks.chunks import chunk_by_paragraph
+
+GROUND_TRUTH = {
+    "whole_text": [
+        {
+            "text": "This is example text. It contains multiple sentences.",
+            "word_count": 8,
+            "cut_type": "paragraph_end",
+        },
+        {
+            "text": "This is a second paragraph. First two paragraphs are whole.",
+            "word_count": 10,
+            "cut_type": "paragraph_end",
+        },
+        {
+            "text": "Third paragraph is a bit longer and is finished with a dot.",
+            "word_count": 12,
+            "cut_type": "sentence_end",
+        },
+    ],
+    "cut_text": [
+        {
+            "text": "This is example text. It contains multiple sentences.",
+            "word_count": 8,
+            "cut_type": "paragraph_end",
+        },
+        {
+            "text": "This is a second paragraph. First two paragraphs are whole.",
+            "word_count": 10,
+            "cut_type": "paragraph_end",
+        },
+        {
+            "text": "Third paragraph is cut and is missing the dot at the end",
+            "word_count": 12,
+            "cut_type": "sentence_cut",
+        },
+    ],
+}
+
+INPUT_TEXT = {
+    "whole_text": """This is example text. It contains multiple sentences.
+    This is a second paragraph. First two paragraphs are whole.
+    Third paragraph is a bit longer and is finished with a dot.""",
+    "cut_text": """This is example text. It contains multiple sentences.
+    This is a second paragraph. First two paragraphs are whole.
+    Third paragraph is cut and is missing the dot at the end""",
+}
+
+
+def run_chunking_test(test_text, ground_truth):
```

Comment:
`ground_truth` in this case is a list of chunks, so let's name it `expected_chunks`.